skins/oasis/css/core/ads-porvata.scss

```diff
@@ -0,0 +1,282 @@
+@import '../../../../extensions/wikia/ArticleVideo/styles/video-shared-variables';
+
+$wikia-top-ads-index: 2;
+
+$default-progress-bar-color: #b7b7b7;
+$default-progress-bar-current-time-color: #666;
+$outstream-progress-bar-color: #666;
+$outstream-progress-bar-current-time-color: #b7b7b7;
+$video-player-button-size: 24px;
+
+$floating-ad-margin: 20px;
+$floating-ad-index: $wikia-top-ads-index + 10;
+$floating-ad-video-width: 320px;
+$floating-ad-animation-speed: $video-collapse-animation-speed;
+$floating-ad-featured-video-gap: $video-container-width;
+
+$vpaid-z-index: 10;
+$video-interactive-area-z-index: $vpaid-z-index + 1;
+
+@mixin absolute-fill() {
+	height: 100%;
+	left: 0;
+	position: absolute;
+	width: 100%;
+	top: 0;
+}
+
+.wikia-ad {
+	.video-overlay {
+		@include absolute-fill();
+	}
+
+	.video-display-wrapper {
+		background: #000;
+		height: 100%;
+		position: relative;
+		width: 100%;
+
+		video, &.vpaid-enabled > div {
+			@include absolute-fill();
+		}
+
+		&.vpaid-enabled video {
+			z-index: $vpaid-z-index;
+		}
+
+		.video-player {
+			background: #000;
+			margin: 0 auto;
+			pointer-events: auto;
+		}
+	}
+
+	.replay-overlay {
+		background: rgba(0,0,0,0.3) url('/extensions/wikia/AdEngine/js/video/player/ui/images/play_button.svg') center/8% no-repeat;
```

Comment:
Maybe we could inline this asset to avoid additional HTTP request? (`/* inline */` makes the deal here)